JesusFilm/core
# refactor: journeys queries

# Description

**Note: only merge after https://github.com/JesusFilm/core/pull/2827!**

### Issue

Components needed for the resources hub need refactored out into the shared space to be used by both journeys-admin and watch. 

We need to refactor out get journey and get journeys queries because they are needed by TemplateSections and TemplateView which will be refactored out in a future PR.

[Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/37663178/todos/7452920587)

### Solution

This PR refactors the getJourney and getJourneys queries into the journeys/ui/lib space. 

# External Changes

n/a

# Additional information

Big-bang discovery branch is here https://github.com/JesusFilm/core/pull/2810.

## How To Test
There should be no behavioural impact so ensure application still performs as expected. These components are currently being used in journeys-admin, journeys, nexus-admin, & watch so regression test as appropriate.
One should be able to use the getJourney query in any project.
One should be able to use the getJourneys query in any project.
